vm_setup_vars.yml

```diff
@@ -37,14 +37,20 @@ baremetal_network_cidr_v6: "{{ lookup('env', 'EXTERNAL_SUBNET_V6') }}"
 baremetal_network_cidr: "{{ baremetal_network_cidr_v4 | default(baremetal_network_cidr_v6, true) }}"
 
 dns_dualstackhost:
-  - ip: "{{ baremetal_network_cidr_v6 | nthhost(5) }}"
+  - ip: "{{  baremetal_network_cidr | nthhost(1) if lookup('env', 'PLATFORM_TYPE') == 'none' else baremetal_network_cidr | nthhost(5) }}"
     hostnames:
       - "api"
 
-dns_extrahosts:
-  - ip: "{{ baremetal_network_cidr | nthhost(5) }}"
+dns_extrahosts: 
+  - ip: "{{ baremetal_network_cidr | nthhost(1) if lookup('env', 'PLATFORM_TYPE') == 'none' else baremetal_network_cidr | nthhost(5) }}"
     hostnames:
       - "api"
+  - ip: "{{ baremetal_network_cidr | nthhost(1) if lookup('env', 'PLATFORM_TYPE') == 'none' else baremetal_network_cidr | nthhost(5) }}"
+    hostnames:
+      - "api-int"
+  - ip: "{{ baremetal_network_cidr | nthhost(1) if lookup('env', 'PLATFORM_TYPE') == 'none' else baremetal_network_cidr | nthhost(4) }}"
+    hostnames:
+      - "*.apps"
```

Comment:
See previous comment

Reply:
I think the concern is this change will affect non-agent dev-script users. Will move adding dns entries inside the agent scripts.